pkg/controller/controller.go

```diff
@@ -145,13 +147,13 @@ func (c *Controller) ProcessNextPodItem() bool {
 // processPodItem verifies that the passed in pod is genuine and, if so, annotates the Phase
 // of the pod in the PVC to indicate the status of the import process.
 func (c *Controller) processPodItem(pod *v1.Pod) error {
-	glog.Infof("processPodItem: processing pod named %q\n", pod.Name)
+	glog.V(Vdebug).Infof("processPodItem: processing pod named %q\n", pod.Name)
 
 	// First get the pod's CDI-relative pvc name
 	var pvcKey string
 	for _, vol := range pod.Spec.Volumes {
 		if vol.Name == DataVolName {
-			glog.Infof("processPodItem: Pod has volume matching CDI claim")
+			glog.V(Vadmin).Infof("processPodItem: Pod has volume matching CDI claim")
```

Comment:
Since we're exposing it for `Vadmin`, could use the pod name in the log line

Reply:
Agree.